hw2/04-api.js

```diff
@@ -1,7 +1,57 @@
-/** Exercise 04 - API **/
+/* eslint-disable no-console */
+/** Exercise 04 - API * */
 
 const url = 'https://restcountries.com/v3.1/all';
 
 // Add your code here
 
+// Function to help format large numbers
+const nf = Intl.NumberFormat();
+
+// Adds a single element to the dom
+const addToDom = (toPrint) => {
```

Comment:
I love that you put all the DOM functionalities into their own functions, it makes it easier to follow your code.